sql/moz-fx-data-shared-prod/firefox_desktop/dataset_metadata.yaml

```diff
@@ -8,3 +8,5 @@ workgroup_access:
 - role: roles/bigquery.dataViewer
   members:
   - workgroup:mozilla-confidential
+  - workgroup:pocket/prefect
+  - workgroup:mozsocial/prefect
```

Comment:
```suggestion
```

Reply:
For the `firefox_desktop` namespace, because this dataset contains workgroup-confidential tables, dataset-level access to all pings shouldn't be provisioned. So to match the access that the pocket prefect account has we don't need to add dataset level access here as long as it's added to the newtab_live table `metadata.yaml` and the cloudops-infra PR.